PodcastGenerator/admin/pg_users.php

```diff
@@ -128,8 +128,10 @@
         }
         ?>
         <?php
-        error:
-        echo '<p style="color: red;">' . $error . '</p>';
+        if(isset($error) && $error) {
+            error:
```

Comment:
I'm curios if this will get detected by the `goto` statement then if it is in a nested statement

Reply:
i’ll test that

Reply:
It occurs to me that I need to switch the order, so line 132 should be flipped with 131, so that the goto exists even if there is not an error, then we will check inside the declaration for an error.